web/src/main/java/com/zebrunner/reporting/web/WebConfig.java

```diff
@@ -164,4 +173,17 @@ public CommonsMultipartResolver multipartResolver() {
         return new CommonsMultipartResolver();
     }
 
+    // TODO: 3/24/20 got rid of the block if jackson.serialization.write-dates-as-timestamps will be false
+    public ObjectMapper objectMapper(ObjectMapper objectMapper) {
+        SimpleModule simpleModule = new SimpleModule();
+        simpleModule.addSerializer(OffsetDateTime.class, new JsonSerializer<>() {
+            @Override
+            public void serialize(OffsetDateTime offsetDateTime, JsonGenerator jsonGenerator, SerializerProvider serializerProvider) throws IOException {
+                jsonGenerator.writeString(DateTimeFormatter.ISO_OFFSET_DATE_TIME.format(offsetDateTime));
+            }
+        });
+        objectMapper.registerModule(simpleModule);
```

Comment:
It seems like you can register the `JavaTimeModule` instead of custom implementation. But before applying this module, you should check if it serializes in correct form.

Reply:
not suitable